conf/code_lists/chief_errors.csv

```diff
@@ -0,0 +1,25 @@
+E3481,decoder.chiefError.MucrAlreadyShut
```

Comment:
why decoder prefix ?

Reply:
What do you think about `error.chief.MucrAlreadyShut` - tells more what it is and where it comes.

Reply:
👍